qulice-checkstyle/src/main/resources/com/qulice/checkstyle/checks.xml

```diff
@@ -344,7 +344,7 @@
         </module>
         <module name="LineLength">
             <property name="max" value="80"/>
-            <property name="ignorePattern" value="^import .*$"/>
+            <property name="ignorePattern" value="^import .*$|https?:\/\/.+"/>
```

Comment:
@JimDeanSpivey this will also ignore lines that have `http://` anywhere, not only in javadoc, something like `\w+* https?:\/\/.+` would be better